README.md

```diff
@@ -26,17 +26,27 @@ With browserify:
 ```
 npm install -D browserify
 
-browserify -p cmify/plugin src/index.js
+browserify -p cmify/plugin -t cmify/transform src/index.js
```

Comment:
If this is merged & `npm publish`'d, will need to also update joshwnj/cmify-example to use the new syntax.